cmd/earthly/main.go

```diff
@@ -1179,9 +1179,48 @@ Set up a whole custom git repository for a server called example.com, using a si
 		{
 			Name: "preview",
 			Subcommands: []*cli.Command{
+				{
+					Name:        "org",
+					Aliases:     []string{"orgs"},
+					Usage:       "Earthly organization administration *experimental*",
+					Description: "Earthly organization administration *experimental*",
+					Subcommands: []*cli.Command{
+						{
+							Name:        "invite",
+							Usage:       "Invite accounts to your organization",
+							Description: "Invite accounts to your organization",
+							UsageText:   "earthly org invite <email> --name <recipient-name> --org <organization-name> --permission <permission> --message <message>",
```

Comment:
In `earthly preview project` the `--org` flag is passed to the `project` command, whereas in `earthly preview org invite`, the `--org` flag is passed to the `invite` command. Should we make this more consistent?

Reply:
I was actually just revising this along with the default org logic.